magda-storage-api/src/test/storage.spec.ts

```diff
@@ -175,6 +177,34 @@ describe("Storage API tests", () => {
                 );
             });
         });
+
+        describe("Upload from the browser", () => {
+            it("Upload some files", () => {
+                return mockAuthorization(
+                    authApiUrl,
+                    true,
+                    jwtSecret,
+                    request(app)
+                        .post("/v0/upload/" + bucketName)
+                        .field("originalname", "test-browser-upload-1")
+                        .attach("image", "src/test/test_image.jpg")
+                        .attach("text", "src/test/test_csv_1.csv")
+                        .expect(200)
+                );
+            });
+
+            it("Upload no files", () => {
+                return mockAuthorization(
+                    authApiUrl,
+                    true,
+                    jwtSecret,
+                    request(app)
+                        .post("/v0/upload/" + bucketName)
+                        .field("originalname", "test-browser-upload-1")
+                        .expect(400)
+                );
+            });
+        });
```

Comment:
Could use a test to ensure that authorisation is required too :)

Reply:
Added